plugins/vli-usbhub/fu-vli-usbhub-device.c

```diff
@@ -691,6 +696,65 @@ fu_vli_usbhub_device_probe (FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_vli_usbhub_device_setup_children (FuVliUsbhubDevice *self, GError **error)
+{
+	FuVliUsbhubPdHdr hdr = { 0x0 };
+	g_autoptr(FuDevice) dev = fu_device_new ();
```

Comment:
This looks like a memory leak that you set dev twice

Reply:
Good catch, thanks!